### PR TITLE
Adding log4j 2.17.1 version

### PR DESCRIPTION
**Pull request by jorgeramirezcarrasco-rovio**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,6 +54,7 @@
         <junit-jupiter.version>5.6.0</junit-jupiter.version>
         <junit.version>4.12</junit.version>
         <junit-platform.version>1.0.1</junit-platform.version>
+        <log4j.version>2.17.1</log4j.version>
```
Comment (juhoautio-rovio) [inline pom.xml:57]: This doesn't have any effect if not referenced. We don't need to add this.

Could you instead set that version on this row:
https://github.com/rovio/rovio-ingest/blob/28f56c2e6299ada66e955cdebbdb81968fa3aa3c/pom.xml#L71

**Comment by jorgeramirezcarrasco-rovio** [comment]
```
mvn dependency:tree | grep log4j                     
[INFO] +- org.apache.logging.log4j:log4j-core:jar:2.17.1:compile
[INFO] |  \- org.apache.logging.log4j:log4j-api:jar:2.17.1:compile
[INFO] |  +- log4j:log4j:jar:1.2.17:compile
[INFO] |  |  |  +- org.apache.logging.log4j:log4j-jul:jar:2.5:compile
[INFO] |  |  |  +- org.apache.logging.log4j:log4j-1.2-api:jar:2.5:compile
[INFO] |  |  +- org.slf4j:slf4j-log4j12:jar:1.7.10:provided

```

File: pom.xml (hunk 1 above)
Comment (juhoautio-rovio) [inline pom.xml:57]: Could you delete this line now?

File: pom.xml [hunk 2]
```diff
@@ -67,7 +68,7 @@
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
-            <version>[2.15.0,)</version>
+            <version>2.17.1</version>
```
Comment (juhoautio-rovio) [inline pom.xml:71]: Could you make the same change in https://github.com/rovio/rovio-ingest/blob/78b412d4c6871021d9738182fbd16d728a81441e/examples/rovio-ingest-maven-example/pom.xml#L49

**Pull request merged**